depends/Makefile

```diff
@@ -165,6 +165,9 @@ $(host_prefix)/share/config.site: check-packages
 
 check-packages: check-sources
 
+clean:
+	@rm -rf work/ built/ sources/ x86_64* i686* mips* arm* aarch64*
```

Comment:
Concept ACK, but having to special-case architectures shows a problem in naming.

Reply:
Agreed that it's less than ideal to need to list all supported architectures here..

If we agree that changing where the architecture-specific stuff ends up is for another PR, what would the ideal directory structure look like here?

`depends/arch/armv7l-unknown-linux-gnueabihf/...`?

(This line could after such a change then `rm -rf arch`.)

Reply:
Sound like a good idea to me, but would like @theuni's opinion on this.